core/core_integrations/grunt.js

```diff
@@ -0,0 +1,125 @@
+var shim = require.once('../shim.js'),
+    server = require.once('http').createServer();
+    io = require.once('socket.io')(server),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/theprojectshop/Kassy/file/2532349301/issues/source?bid=3282987&fileBranchId=3283883#l3)